src/Routing/Router.php

```diff
@@ -1015,6 +1015,7 @@ public static function parseRoutePath(string $url): array
             (?<controller>[a-z0-9]+)
             ::
             (?<action>[a-z0-9_]+)
+            (?<params>(?:/(?:[a-z][a-z0-9-_]*=)?[a-z0-9-_]+)+/?)?
```

Comment:
```suggestion
            (?<params>(?:\/(?:[a-z][a-z0-9-_]*=)?(["']?[a-z0-9-_]+['"]?))+\/?)?
```

I think this makes it possible to parse the quotes out. It will require slightly more complex parsing logic. I'm happy to take care of that if you want.

Reply:
I see what you mean but if we want to support something like
```
Bookmarks::view/organization="cakephp=test"
```
then it should be this regex in my opinion
```
(?<params>(?:/(?:[a-z][a-z0-9-_]*=)?(["\']?[a-z0-9-_=]+[\'"]?))+/?)?
```
But as you already said the splitting of `key=value` is more complex... Need to look into it.